### PR TITLE
Add Learning-PETs materials to SecretFlow

### PR DESCRIPTION
**Pull request by Jamie-Cui**

### CODE REVIEW

**Review by zhouaihui — COMMENTED** [review]
Awesome work!

**Review by anakinxc — DISMISSED** [review]
Nice

**Review by zhouaihui — APPROVED** [review]

**Review by zhouaihui — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -46,6 +46,13 @@ Please check [INSTALLATION.md](./docs/getting_started/installation.md)
 
 Please check [DEPLOYMENT.md](./docs/getting_started/deployment.md)
 
+## Learn PETs
```
Comment (zhouaihui) [inline README.md:49]: Add this to `README.zh-CN.md` also.

Comment (Jamie-Cui) [reply README.md:49]: Done

**Review by yingting6 — APPROVED** [review]
LGTM

**Pull request merged**